edge-app-template/src/clj/new/edge_app_template.clj

```diff
@@ -50,6 +57,13 @@
                    ["src/{{name}}.scss" (render "app.css" data)])
           (->files data
                    ["src/public/{{name}}.css" (render "app.css" data)]))
+        (when (:reagent data)
```

Comment:
you've got conflicts with `--cljs` below, I think it should be valid to pass:

```
./bin/app --cljs --reframe
```

and that should not break things

Reply:
They are different apps, I don't think a user should expect a different result from `-- cljs --reframe` to just `--reframe`.